src/posts/PublishedDate.js

```diff
@@ -3,7 +3,10 @@ import React from 'react';
 class PublishedDate extends React.Component{
     render(){
         return(
-            <p>{this.props.publishedDate.toString()}</p>
+            <div className="left-align-board">
+                <p>{this.props.publishedDate.toString()}</p>
```

Comment:
`publishedDate` is a Javascript Date object. Javascript Date object has many function that prints out the date in different format. `toString()` is probably not the function we want to use. Consider other functions, after reviewing https://www.w3schools.com/jsref/jsref_todatestring.asp and other functions.